lte/gateway/c/oai/tasks/sgw_s8/sgw_s8_handlers.c

```diff
@@ -12,15 +12,229 @@ limitations under the License.
 */
 #include <stdio.h>
 #include <stdint.h>
+#include <stdlib.h>
 #include "log.h"
 #include "common_defs.h"
-#include "intertask_interface.h"
+#include "sgw_context_manager.h"
+#include "spgw_types.h"
+#include "sgw_s8_state.h"
+#include "sgw_s8_s11_handlers.h"
+#include "s8_client_api.h"
 
-int sgw_s8_handle_s11_create_session_request(
+uint32_t sgw_get_new_s1u_teid(sgw_state_t* state) {
+  if (state->s1u_teid == 0) {
+    state->s1u_teid = INITIAL_SGW_S8_S1U_TEID;
+  }
+  __sync_fetch_and_add(&state->s1u_teid, 1);
+  return state->s1u_teid;
+}
+
+uint32_t sgw_get_new_s5s8u_teid(sgw_state_t* state) {
+  __sync_fetch_and_add(&state->s5s8u_teid, 1);
+  return (state->s5s8u_teid);
+}
+
```

Comment:
Add comment for spgw_ue_context_t; Re-using the spgw_ue context structure, that contains the list sgw_s11_teids and is common across both sgw_s8 and spgw tasks.